sources/generate-readme/src/lib.rs

```diff
@@ -84,6 +84,9 @@ where
     )
     .map_err(|e| error::ReadmeGenerateSnafu { error: e }.build())?;
 
+    // Make sure the end of the file has a newline
+    content = content + "\n";
```

Comment:
Should we double-check before we add a newline? Maybe it's not necessary if we expect cargo readme to always trim it...

```suggestion
    if content.chars().last().unwrap_or_default() != '\n' {
        content = content + "\n";
    }
```

Reply:
Sure 👍